custom/icds_reports/migrations/sql_templates/database_views/service_delivery_report_view.sql

```diff
@@ -1,25 +1,25 @@
 DROP VIEW IF EXISTS service_delivery_report CASCADE;
 CREATE VIEW service_delivery_report AS
 SELECT
-"awc_location_months"."awc_id" AS "awc_id",
-"awc_location_months"."awc_name" AS "awc_name",
-"awc_location_months"."awc_site_code" AS "awc_site_code",
-"awc_location_months"."supervisor_id" AS "supervisor_id",
-"awc_location_months"."supervisor_name" AS "supervisor_name",
-"awc_location_months"."supervisor_site_code" AS "supervisor_site_code",
-"awc_location_months"."block_id" AS "block_id",
-"awc_location_months"."block_name" AS "block_name",
-"awc_location_months"."block_site_code" AS "block_site_code",
-"awc_location_months"."district_id" AS "district_id",
-"awc_location_months"."district_name" AS "district_name",
-"awc_location_months"."district_site_code" AS "district_site_code",
-"awc_location_months"."state_id" AS "state_id",
-"awc_location_months"."state_name" AS "state_name",
-"awc_location_months"."state_site_code" AS "state_site_code",
-"awc_location_months"."block_map_location_name" AS "block_map_location_name",
-"awc_location_months"."district_map_location_name" AS "district_map_location_name",
-"awc_location_months"."state_map_location_name" AS "state_map_location_name",
-"awc_location_months"."aggregation_level" AS "aggregation_level",
+"awc_location_local"."doc_id" AS "awc_id",
```

Comment:
changing the table here seems fine, but curious why you are doing it

Reply:
that was mostly just because  awc_location_months  does not `<location>_is_test` columns. But I can revert it since I will be filtering them in the fetching layer

Reply:
Well I still need this awc_location_local as I need this <location>_is_test as column of this view